sdk/compute/Microsoft.Azure.Management.Compute/src/Properties/AssemblyInfo.cs

```diff
@@ -7,9 +7,9 @@
 [assembly: AssemblyTitle("Microsoft Azure Compute Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Compute Resources.")]
 
-[assembly: AssemblyVersion("54.0.0")]
-[assembly: AssemblyFileVersion("54.0.0")]
-[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyVersion("54.1.0")]
```

Comment:
Keep the `AssemblyVersion`, unless it's a major change. See: https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=signing-and-publishing-nuget-packges